themes/docs-new/layouts/shortcodes/EOL_chefdk.md

```diff
@@ -0,0 +1,9 @@
+<div class="admonition-warning"><p class="admonition-warning-title">Warning</p><div class="admonition-warning-text">
+
+ChefDK is [deprecated](/versions/#deprecated-products-and-versions) and no
```

Comment:
There's no need to contact an account rep. Users should download Chef Workstation, which includes all the functionality of ChefDK and a whole lot more.